src/components/App/addressForm.js

```diff
@@ -1,100 +1,115 @@
-import React from 'react';
-import { Row, Col, Alert, Input, Button } from 'antd';
-import { get } from 'axios';
+import React from "react";
+import { Row, Col, Alert, Input, Button } from "antd";
+import { get } from "axios";
 
 const faucetStatuses = [
-  'unchecked',
-  'requesting',
-  'generating',
-  'funded',
-  'rejected',
-  'error',
-]
+  "unchecked",
+  "requesting",
+  "generating",
+  "funded",
+  "rejected",
+  "error"
+];
 
 export default class AddressForm extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       status: faucetStatuses[0],
-      address: '',
-      generatedAddress: '',
-      wif: '',
-      tx: '',
+      address: "",
+      generatedAddress: "",
+      wif: "",
+      tx: ""
     };
   }
 
   handleAddress = e => {
-    const address = e.target.value
-    console.log(address);
+    const address = e.target.value;
     this.setState({ address });
-  }
+  };
 
   request = async () => {
-    this.setState({ status: faucetStatuses[1] })
+    this.setState({ status: faucetStatuses[1] });
     try {
-      const { status, data } = await get(`${this.props.config.faucetSite}/request/${this.state.address}`);
+      const { status, data } = await get(
+        `${this.props.config.faucetSite}/request/${this.state.address}`
+      );
       if (status === 200) {
         this.setState({ status: faucetStatuses[3], tx: data });
       } else {
         this.setState({ status: faucetStatuses[4] });
       }
-    } catch(e) {
+    } catch (e) {
       this.setState({ status: faucetStatuses[5] });
     }
-  }
+  };
 
   generate = async () => {
-    this.setState({ status: faucetStatuses[2] })
-    const { status, data } = await get(`${this.props.config.faucetSite}/generate`);
+    this.setState({ status: faucetStatuses[2] });
+    const { status, data } = await get(
+      `${this.props.config.faucetSite}/generate`
+    );
     if (status === 200) {
       this.setState({ status: faucetStatuses[3], ...data });
     } else {
       this.setState({ status: faucetStatuses[5] });
     }
-  }
+  };
 
   render() {
     let successMessage;
     if (this.state.status === faucetStatuses[3]) {
-      successMessage = this.state.wif !== ''
-        ? `Congratulations! Your address is ${this.state.generatedAddress} and your WIF is ${this.state.wif}, look for the following tx: ${this.state.tx}. Happy developing!`
-        : `Congratulations, look for the following tx: ${this.state.tx}. Happy developing!`;
+      successMessage =
+        this.state.wif !== ""
+          ? `Congratulations! Your address is ${
```

Comment:
A bit opinionated, but why are we congratulating people? :smile:

Reply:
Ah, congratulating them on using the faucet? Which makes creating an account on privatenet lightning fast?

Something like that 😛